engine/configurator/conf_yaml.c

```diff
@@ -28,21 +27,128 @@
 #define YAML_NODE_LINE_COLUMN(_n)   \
     (_n)->start_mark.line + 1, (_n)->start_mark.column + 1
 
-#define YAML_TARGET_CONTEXT_INIT \
-    { .oid = NULL, \
-      .value = NULL, \
-      .access = NULL, \
-      .type = NULL, \
-      .xmlvolatile = NULL, \
-      .substitution = NULL, \
-      .unit = NULL, \
-      .deps = SLIST_HEAD_INITIALIZER(deps), \
-      .cond = TRUE }
+typedef enum cs_yaml_node_type {
```

Comment:
Is there a reason that these types and mappings have `cs_` and not `cfg_` prefix? And being file-local, do they need a prefix at all?

Reply:
I think there is no any reason for using prefix in such a case. Is there a reason to eliminate prefixes? Should I do it?